test/cypress/integration/04-organization_v2.test.js

```diff
@@ -3,17 +3,34 @@ describe('New organization profile', () => {
    * Contributions section is already tested in `05-user_v2.test.js`
    * About section is already tested in `04-collective_v2.test.js`
    */
+  before(() => {
+    cy.createCollective({ type: 'ORGANIZATION' }).then(collective => {
+      const collectiveSlug = collective.slug;
 
+      cy.visit(`/${collectiveSlug}/`);
+    });
+  });
   describe('Contributors section', () => {
-    it.skip('Only shows core contributors without any filter', () => {
-      // TODO
+    it('Only shows core contributors without any filter', () => {
+      cy.getByDataCy('section-contributors').click({ force: true });
+      cy.hash().should('eq', '#section-contributors');
```

Comment:
Great that you checked that!

Reply:
Hey @Betree  I need more clarity on the changes required

Reply:
No changes required, I was just saying that this was a great change :-)